opcua/common/ua_utils.py

```diff
@@ -143,3 +144,19 @@ def _get_node_supertypes(node):
        basetypes.extend( _get_node_supertypes(parents[0]) )
        
     return basetypes
+
+def is_child_present(node, browsename):
+    """
+    return if a browsename is present a child from the provide node
+    :param node: node wherein to find the browsename
+    :param browsename: browsename to search
+    :returns returne True if the browsename is present else False 
+    """
+    is_present = True
+    #TODO: Is there a better way to test if a browse name is already present then with an exception?                    
```

Comment:
yes do one query: node.get_children_descriptions(xxx):
then check if your browse name is in the response